receiver/prometheusreceiver/README.md

```diff
@@ -90,6 +90,12 @@ prometheus --config.file=prom.yaml
 "--feature-gates=receiver.prometheusreceiver.RemoveLegacyResourceAttributes"
 ```
 
+- `receiver.prometheusreceiver.RemoveStartTimeAdjuster`: If enabled, the prometheus receiver no longer sets the start timestamp of metrics if it is not known. Use the `metricstarttime` processor instead if you need this functionality.
```

Comment:
Internally this doesn't just remove the `StartTimeMetricAdjuster`, so the name feels potentially misleading if someone attempts to diagnose an issue or maintain the code. Perhaps `RemoveStartTimeEstimation` or something else might be clearer?

Reply:
It will remove both the IntialPointAdjuster and the StartTimeMetricAdjuster after it graduates to GA. But we have to keep those around while the deprecation is in progress

Reply:
I could call it `RemoveStartTimeAdjustment` if you would prefer

Reply:
renamed